### PR TITLE
Pom 250 record community decision Early Allocation Part IV

### PR DESCRIPTION
**Pull request by starswan**

### CODE REVIEW

File: app/views/early_allocations/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<%= form_for @early_assignment, url: community_decision_prison_prisoner_early_allocation_path(@prison, @early_assignment.nomis_offender_id), method: :post do |form| %>
+  <% if @early_assignment.errors.count > 0 %>
+    <%= render :partial => "/shared/field_errors", :locals => { :errors => @early_assignment.errors, form: form } %>
+  <% end %>
+
+  <h1 class="govuk-heading-xl govuk-!-margin-top-4">Record a decision from the community probation team about early allocation</h1>
+
+  <%= render partial: "boolean_field", locals: { form: form,
+                                                 heading_text: "What decision did the community probation team make?",
+                                                 hint_text: "If the community agreed to take this case early, a new handover date will be calculated automatically",
```
Comment (rossjones) [inline app/views/early_allocations/edit.html.erb]: Might have missed it, can't see where the handover date changes...

Comment (starswan) [reply app/views/early_allocations/edit.html.erb]: I've just realised that there is a massive hole in this story - there is nothing to tell us how these actions affect the handover dates...

**Comment by KPobeeNorris** [comment]
@starswan - after you update the community decision the prisoner page row for Early Allocation says "Waiting for community decision".  Does the content need updating here, or is it in another PR?

**Comment by starswan** [comment]
> @starswan - after you update the community decision the prisoner page row for Early Allocation says "Waiting for community decision". Does the content need updating here, or is it in another PR?

The community decision is recorded to part V

**Comment by starswan** [comment]
I think that is fine

**Pull request merged**